### PR TITLE
Apply Pricing changes

### PR DESCRIPTION
**Pull request by fjvallarino**
There is some more work to do regarding translations (some wording questions and, ideally, removing unused strings), but the code is ready for review.

@Rise-Vision/apps please review

### CODE REVIEW

File: src/js/components/plans/svc-plan.js [hunk 1]
```diff
@@ -47,56 +108,12 @@
...
-          var deferred = $q.defer();
-          var search = "(productTag=Plans)";
-
-          _factory.getPlans({
-            search: search
-          })
-            .then(function (resp) {
-              $log.debug("getPlansDetails response.", resp);
-
-              return _getSelectedCurrency().then(function (currency) {
```
Comment (alex-deaconu) [inline src/js/components/plans/svc-plan.js]: @fjvallarino to confirm, new plans have the same price regardless of CAD/USD currency? Thanks.

Comment (fjvallarino) [reply src/js/components/plans/svc-plan.js]: @alex-deaconu we discussed with @ezequielc and decided to hardcode the same price at this point, and review in a later release. I thought about leaving the currency logic there, but it was confusing (we can add it back when needed)

Comment (alex-deaconu) [reply src/js/components/plans/svc-plan.js]: @fjvallarino sounds good, thanks!

File: src/templates/components/plans/plans-modal.html [hunk 2]
```diff
@@ -5,81 +5,113 @@
...
+            <div ng-show="!isFree(plan)">
+              <h3>{{plan.proLicenseCount}}</h3>
+              <span ng-show="plan.proLicenseCount === 1" translate>common-header.plans.displayIncluded</span>
+              <span ng-show="plan.proLicenseCount > 1" translate>common-header.plans.displaysIncluded</span>
             </div>
+            <p ng-show="showSavings(plan)" class="planSavings"
+               translate translate-values="{ save: (monthlyPrices ? plan.monthly.save : plan.yearly.save) }">
+              common-header.plans.saveEachYear
+            </p>
+            <p ng-show="showSavings(plan) && isCurrentPlanSubscribed(plan)" translate>
```
Comment (alex-deaconu) [inline src/templates/components/plans/plans-modal.html]: @fjvallarino from what I understand, we don't show the "Need more Displays" wording for the Starter plan? Just making sure it's intentional. Thanks.

Comment (fjvallarino) [reply src/templates/components/plans/plans-modal.html]: @alex-deaconu good catch. I'll update now

Comment (alex-deaconu) [reply src/templates/components/plans/plans-modal.html]: @fjvallarino thanks for fixing!

**Comment by alex-deaconu** [comment]
@fjvallarino looks good 👍 . Tests failed however I think it's a random failure from what I can tell. Thanks.

**Comment by fjvallarino** [comment]
@alex-deaconu I added a bunch of small commits, mainly related to translations (expect the first two of the unreviewed batch)

**Comment by alex-deaconu** [comment]
@fjvallarino looks good 👍

**Pull request merged**